docs/README.md

```diff
@@ -185,16 +185,38 @@ button {
 ```
 
 ## Reset
-Nib comes bundled with Eric Meyer's style reset support, you can choose to apply the global or any specifics that you wish. To view the definitions view [reset.styl](https://github.com/visionmedia/nib/blob/master/lib/nib/reset.styl)
-
-- global-reset()
-- nested-reset()
-- reset-font()
-- reset-box-model()
-- reset-body()
-- reset-table()
-- reset-table-cell()
-- reset-html5()
+Nib comes bundled with [Eric Meyer's style reset](eric-meyer) and [Nicolas Gallagher's _Normalize_](normalize) support and, you can choose to apply the global or any specifics that you wish. To view the definitions view [`reset.styl`](https://github.com/visionmedia/nib/blob/master/lib/nib/reset.styl).
```

Comment:
"visionmedia" actually changed his name to "tj", so that link should probably be fixed